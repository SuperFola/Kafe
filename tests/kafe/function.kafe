fun main() -> int
    x : int = 10
end

fun bis(x: int) -> int
    y: int = 12
    z: string = "test aya"
end

fun ter(x: int, a: string) -> int
    x: int = 10
end
