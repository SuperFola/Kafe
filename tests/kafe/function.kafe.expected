(Program
    (Function (Name main) (Args) (Type int) (Body (Definition (VarName x) (Type int) (Integer 10))))
    (Function (Name bis) (Args (Declaration (VarName x) (Type int))) (Type int) (Body (Definition (VarName y) (Type int) (Integer 12)) (Definition (VarName z) (Type string) (String test aya))))
)